.buildscript/test.sh

```diff
@@ -6,15 +6,15 @@ set -e
 DIR=`pwd`
 trap "cd ${DIR}" SIGINT SIGTERM EXIT
 
-./gradlew clean build publishToMavenLocal --stacktrace --daemon
```

Comment:
Why remove? Redundancy?

Reply:
In CI this is the first command invoked, so it doesn't help there.  Locally, I got tired of rebuilding All The Things as I ran different tests.